### PR TITLE
Update styling for login and registration pages to match 2021 design

### PR DESCRIPTION
**Pull request by rajpiskala**
**HTML:**
- Added title for registration page
- Made titles of the sections uppercase

**CSS:**
- Increased margin between title and form, sign up and google section
- Changed the colors to use var() instead of hardcoded values
- Set font weight of text to bold, increased font size

**Questions/Concerns:**
1. Try stretching the page with the login/registration form. It's jumpy and it resizes to different widths. I believe this is due to Bootstrap aligning to columns but I don't understand the Bootstrap styling well enough to fix this/figure out why it's like this.

### CODE REVIEW

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: Can you get rid of this comment? We don't need it

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: Links on black backgrounds should change to `var(--white)` on hover. Check out the doc pinned in the group chat for all the hover rules

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: This is actually being overshadowed by the `.ideahacks-form p` rule (line 112) because that one has a higher specificity. This means that the error message is always white, and therefore unreadable

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: Nitpick, but for consistency can you change this to `var(--white)`?

File: views/login.hbs
Comment (robertnperalta) [inline views/login.hbs]: Can you change this to be `col-xs-12` and remove the `col-xs-offset-1`? On small screens, it's way too cramped with margins that are too wide for the screen. I think the just padding inside the div will be enough on small screens, but I leave it up to your discretion if you think it needs more

File: views/login.hbs
Comment (robertnperalta) [inline views/login.hbs]: You can just delete this

File: views/login.hbs
Comment (robertnperalta) [inline views/login.hbs]: Move this into the main login box and make the warning smaller. Right now, users might have to scroll down to see that this option exists on certain screens (like mine), and we want to make sure that the convenient option is still front and center. I realize that the button is white on white, so you might have to put a shadow. Alternatively, you can use a different version of the button that you can find [here](https://developers.google.com/identity/branding-guidelines)

File: views/partials/applicationModal.hbs
Comment (robertnperalta) [inline views/partials/applicationModal.hbs]: Do you have your own formatter in your IDE? This file and `views/dashboard-application.hbs` have been formatted, but I don't think it was our Prettier, because no one has edited these files recently and my commits would have cleaned it up already. Either way, I think you'll need to revert these changes to get your build passing

File: views/registration.hbs
Comment (robertnperalta) [inline views/registration.hbs]: Very small nitpick, but the extra space here bugs me

File: views/login.hbs
Comment (robertnperalta) [inline views/login.hbs]: Another small thing, both for this page and the registration page, but you can remove `.theme-yellow` from these since the class doesn't exist anymore (and is a bit misleading now)

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: `var(--light-green)` is actually no longer used in the design, so we shouldn't use it. I pushed the commit to `style.css` before Courtney made the change, but I'll remove it soon. Just switch to `var(--green)`

**Review by robertnperalta — CHANGES_REQUESTED** [review]
Overall, looks good. Don't worry about the jittering as the screen shrinks. You're right, it's Bootstrap stuff, but it's not too egregious and it would take a lot more work to center everything responsively by hand. 

If you scroll down to the bottom of this page, you'll notice that your branch is actually failing its Travis build. I encourage you to click through and take a look at what the build looks like, but the issue is that it doesn't match our Prettier format. Chances are, right now you'll have changes pending in your workspace. This is because Prettier runs after you run `git push`, leaving changes behind. Just commit those changes with the message `Run Prettier` and push them up again.

File: views/registration.hbs
Comment (yolanda10202) [inline views/registration.hbs]: I was wondering if it'll be an easier fix to add text-transform: uppercase; to label in the .css file. Since in future designs the text for label may or may not be all uppercase.

Comment (rajpiskala) [reply views/registration.hbs]: I agree, what you said makes a lot of sense. I've reverted it back to lowercase, thanks!

File: views/partials/applicationModal.hbs
Comment (rajpiskala) [reply views/partials/applicationModal.hbs]: Yeah I do, I didn't realize that Husky ran prettier *after* the push, I thought it would run and amend the commit with the prettified version. Anyways I was able to get the build passing with that, so thanks!

File: views/login.hbs
Comment (rajpiskala) [reply views/login.hbs]: It definitely looks better with `col-xs-12` and without `col-xs-offset-1` on small screens, so I'll go with your suggestion.

File: views/login.hbs
Comment (rajpiskala) [reply views/login.hbs]: I forgot to test it with the mobile view so I didn't catch that, thanks! Should I add @g.ucla.edu emails only for the registration page as well?

Comment (robertnperalta) [reply views/login.hbs]: Nope, the only @g.ucla.edu emails only applies to the Google login button. We take any .edu email for normal accounts

Comment (robertnperalta) [reply views/login.hbs]: Ah, sorry I wasn't clear here. I was talking about the Google login button along with the text. Bring the dividing line along too so it's clear the g.ucla.edu restriction only applies to the Google login button

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: This rule currently applies to the input boxes as well, which makes users think their caps lock is on. I think just removing `.form-control` would fix this, but double-check that it doesn't break anything else.

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: As far as I can tell, you don't need the `!important` here. I guess it won't affect any other pages, so it doesn't really matter, but unnecessary `!important`s make me nervous because they can often mean extra debugging in the future. I leave it up to you if you want to take it out or not; doesn't make a functional difference

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css:101]: Can you add `font-weight: normal` to these rules? Having the input text be bolded looks a bit too chunky

File: views/login.hbs
Comment (robertnperalta) [inline views/login.hbs]: Like I mentioned in the above unresolved comment, can you move the `hr` and button up to the main login section as well? Sorry I wasn't specific before; I always forget GitHub doesn't show exactly which lines you comment on

File: views/registration.hbs
Comment (robertnperalta) [inline views/registration.hbs]: Can you make the `theme-yellow` and `col-xs-12` changes on this div too?

**Review by robertnperalta — CHANGES_REQUESTED** [review]
Just a couple more things to fix this time around, but it's looking good!

File: public/css/login-registration.css
Comment (rajpiskala) [reply public/css/login-registration.css]: Yeah, I definitely agree with !important making debugging a lot harder. I've gotten rid of it to make it easier on future developers in dealing with specificity issues. Thanks for catching that!

File: public/css/login-registration.css
Comment (rajpiskala) [reply public/css/login-registration.css]: This makes the placeholder text capital case, is that fine or should that be uppercase still? Here's a picture for reference. 
Capital case:
image

Uppercase:
image
Personally I think capital case looks better here so I've kept it that way for my commit.

File: views/login.hbs
Comment (rajpiskala) [reply views/login.hbs]: Done, thanks for pointing this out!

File: public/css/login-registration.css
Comment (robertnperalta) [inline public/css/login-registration.css]: Can you delete this comment just to keep everything clean?

**Review by robertnperalta — APPROVED** [review]
Looks great! Just two small cleanup things to do, one of which I left a comment for. The other one is the `.fullscreen` class on the `<html>` tag. I never noticed that before, but it's an old fix that's kind of weird and is now superseded by stuff in `style.css`. Just so it never confuses anyone in the future, can you remove the class from both the HTML and CSS?

My work has a thing that lets you request small changes and then automatically approve it after those changes are done. I wish GitHub would let me do that. Once you push those changes, I'll merge it up. Thanks!

File: public/css/login-registration.css
Comment (robertnperalta) [reply public/css/login-registration.css]: I agree that capital case is better here; there's already a lot of bold/thick/uppercase text, and the capital case is closer to how the user will be typing, making them feel more comfortable.

File: public/css/login-registration.css
Comment (rajpiskala) [reply public/css/login-registration.css]: Done! I checked on VS Code and it looks like class="fullscreen" is only for the login and registration, so we should be good on that. I removed the commented code too.

**Pull request merged**
